src/components/msal-auth-context/msal-auth-provider.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  AuthenticationResult,
+  EventCallbackFunction,
+  EventType,
+  InteractionType,
+  PublicClientApplication,
+} from '@azure/msal-browser';
+import { AuthCodeMSALBrowserAuthenticationProvider } from '@microsoft/microsoft-graph-client/authProviders/authCodeMsalBrowser';
+import { PropsWithChildren, useEffect, useMemo } from 'react';
+
+import { msGraphConfig, serviceNowApiConfig, msalConfig } from './config';
+import { useDispatch } from 'react-redux';
+import { setAccount, setProvider } from '../../store/msal/reducer';
+import ServiceNowAuthProvider from './service-now-auth-provider';
+import { MsalProvider } from '@azure/msal-react';
+
+export type MsalContext = {
+  graphAuthProvider?: AuthCodeMSALBrowserAuthenticationProvider;
+  serviceNowAuthProvider?: ServiceNowAuthProvider;
```

Comment:
```suggestion
  serviceNowAuthProvider?: AuthCodeMSALBrowserAuthenticationProvider;
```
I think we can use the auth provider from MSAL instead of our own. If it works, we can delete the ServiceNowAuthProvider

Reply:
Genialt :D, funka som fjell det!